src/app/pages/Swap/modals/SwapSelectAsset/index.tsx

```diff
@@ -60,20 +119,7 @@ export const SwapSelectAssetModal = memo<SelectTokenModalProps>(
           />
         </div>
 
-        <div className="px-4 pb-4 flex-1 flex flex-col overflow-y-auto">
-          <Suspense fallback={<SpinnerSection />}>
-            {accountTezAddress && (
-              <TezosChainAssetsList
-                chainId={network.chainId}
-                route3tokensSlugs={route3tokensSlugs}
-                filterZeroBalances={inputName === 'input'}
-                publicKeyHash={accountTezAddress}
-                searchValue={searchValueDebounced}
-                onAssetSelect={handleAssetSelect}
-              />
-            )}
-          </Suspense>
-        </div>
+        <Suspense fallback={<SpinnerSection />}>{AssetsList}</Suspense>
```

Comment:
`Spinner` component is about to be removed from all UIs. Please use `Loader` or `PageLoader` instead